ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -562,6 +619,9 @@ struct FrameworkBuilder {
     for platform in Architecture.TargetPlatform.allCases {
       // Get all the slices that belong to the specific platform in order to lipo them together.
       let slices = thinArchives.filter { $0.key.platform == platform }
+      if slices.count == 0 {
```

Comment:
Very minor in this context. In general, it is recommended to use [isEmpty](https://developer.apple.com/documentation/swift/array/1688398-isempty) method instead of `count == 0` or similar for collections because `count` may be non-constant time for some collection implementations.

Reply:
Thanks - I'll fix for next PR